mods/ts/mod.yaml

```diff
@@ -20,7 +20,7 @@ MapFolders:
 
 Packages:
 # Tiberian Sun
-	~scores.mix:CRC32
+	~SCORES.MIX:CRC32
```

Comment:
:-1: This will clash with the filename when downloading. https://github.com/OpenRA/OpenRAWeb/blob/master/content/packages/ts-music-mirrors.txt